addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -221,7 +222,10 @@ export class LinkPopover extends Component {
             if (icon) {
                 this.state.previewIcon.value = icon;
             }
-        } else if (window.location.hostname !== url.hostname) {
+        } else if (
+            window.location.hostname !== url.hostname &&
+            !new RegExp(`^https?://${session.db}\\.odoo\\.com(/.*)?$`).test(url.origin)
+        ) {
```

Comment:
This doesn't seem to be part of either of the two commits mentionned in the commit message?

Reply:
No, no, I didn't mean the import. I meant this if condition. But I was wrong, it is in the second commit indeed. I don't know how I didn't see it yesterday, my bad.